app/components/trainee_name/view.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module TraineeName
+  class View < GovukComponent::Base
+    attr_reader :trainee
+
+    def initialize(trainee)
+      @trainee = trainee
+    end
+
+    def display_text
+      full_name || t("views.trainees.show.draft")
```

Comment:
I don't think we'd want default text of `Draft` for non drafts.

Reply:
@edwardhorsford 
Would we have any non-draft trainees that don't have name set?